playground/src/style.scss

```
@@ -1,9 +1,9 @@
-@import "../../assets/stylesheets/colors";
-@import "../../assets/stylesheets/variables";
-@import "../../assets/stylesheets/mixins";
-@import "../../assets/stylesheets/breakpoints";
-@import "../../assets/stylesheets/animations";
-@import "../../assets/stylesheets/z-index";
+@import "../../packages/base-styles/colors";
```

Minor: Was the `~` prefix an option? That would match the documentation.

```suggestion
@import "~@wordpress/base-styles/colors";
```

---

Good idea! Updated in https://github.com/WordPress/gutenberg/pull/17883/commits/580909af0f472e15a12b095d02a28c193ca81ccc